src/custom/components/Tokens/TokensTable.tsx

```diff
@@ -220,7 +249,7 @@ export default function TokenTable({
         </AutoColumn>
       ) : (
         <LoadingRows>
-          {Array.from(Array(maxItems * 4), (_, i) => (
+          {Array.from(Array(loadingRows * 6), (_, i) => (
```

Comment:
don't use magic numbers, maybe better to have this "6" cache somewhere

Reply:
Done